### PR TITLE
Add five new books

### PR DESCRIPTION
**Pull request by ibrahimraimi**
# Pull Request
Added 5 new books

## Description
- Add [MongoDB Notes for Professionals](https://goalkicker.com/MongoDBBook/) to [Database](https://github.com/EOEboh/Free-Software-Books/blob/main/books/dev-books/Database/database.md)
- Add [JavaScript for Beginners](https://drive.google.com/file/d/1ZS0yIqrtFq1EzJcMb_v5mr_OXP5z3POt/view) to [JavaScript](https://github.com/EOEboh/Free-Software-Books/blob/main/books/dev-books/JavaScript/javascript.md)
- Add [TypeScript Deep Dive](https://www.programmer-books.com/wp-content/uploads/2018/08/typescript.pdf) to [TypeScript](https://github.com/EOEboh/Free-Software-Books/blob/main/books/dev-books/TypeScript/typescript.md)
- Add [HTML5 Canvas Notes for Professionals](https://goalkicker.com/HTML5CanvasBook/) to [HTML and CSS](https://github.com/EOEboh/Free-Software-Books/blob/main/books/dev-books/HTML%20and%20CSS/htmlcss.md)
- Add [Python Notes for Professionals](https://goalkicker.com/PythonBook/) and [Introduction to Scientifc Programming with Python](https://library.oapen.org/bitstream/id/56d27e73-e92a-4398-8198-239be7aacc93/2020_Book_IntroductionToScientificProgra.pdf) to [Python](https://github.com/EOEboh/Free-Software-Books/blob/main/books/dev-books/Python/python.md)

## Related Issue

No opened issues

### CODE REVIEW

File: books/dev-books/Database/database.md [hunk 1]
```diff
@@ -7,7 +7,13 @@ Please kindly follow this structure when you are contributing
...
+- [Title of the book pdf](direct link to the book)
+
 ### P
 
 - [PostgreSQL](https://goalkicker.com/PostgreSQLBook/)
 - [Practical SQL: A beginners Guide](https://drive.google.com/file/d/1LGcFIlyeYUz0zxghNHlj9KQ7azCGX90G/view?usp=share_link)
+
+### M
+
+- [MongoDB Notes for Professionals](https://goalkicker.com/MongoDBBook/)
```
Comment (binadiegha) [inline books/dev-books/Database/database.md:19]: Please adjust the alphabetical order by moving the **###M** section above the **###P** section

File: books/dev-books/HTML and CSS/htmlcss.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ Please kindly follow this structure when you are contributing
 
 ### C
 - [CSS3 The Missing Manual](https://drive.google.com/file/d/1L3tVGwLb_ZMbdfq7gmV75Ixl96CuOty0/view?usp=share_link)
+- [CSS Notes for Professionals](https://goalkicker.com/CSSBook/)
 
 - [CSSS SECRETS: BETTER SOLUTIONS TO EVERYDAY CSS PROBLEMS (INTERMEDIATE LEVEL)](https://www.pdfdrive.com/css-secrets-better-solutions-to-everyday-web-design-problems-e177990723.html)
```
Comment (binadiegha) [inline books/dev-books/HTML and CSS/htmlcss.md:17]: If you could help us add a *###H* section in your next PR, it would be super awesome 😎

**Review by binadiegha — CHANGES_REQUESTED** [review]
@ibrahimraimi  Thanks for the Beautiful books added. We appreciate your efforts and hope you contribute more🚀

File: books/dev-books/Database/database.md (hunk 1 above)
Comment (ibrahimraimi) [reply books/dev-books/Database/database.md]: Oh, my bad.
fixing the problem now.

**Review by binadiegha — APPROVED** [review]
Awesome, LGTM 🚀 thanks for your contributions 🚀😁💪🏾

**Comment by binadiegha** [comment]
@ibrahimraimi please could you resolve the conflicts?

**Comment by binadiegha** [comment]
Thank you 💪🏾

**Pull request merged**